.registry/app.yaml

```diff
@@ -1,11 +1,11 @@
-title: "Minimal Azure Environment"
+title: "Stack Azure Sample"
```

Comment:
`stack-azure-sample` sounds OK for repo name but `Stack Azure Sample` sounds awkward for a title. It's `Azure Sample Stack` in README. @hasheddan do you mind if I update it to be `Azure Sample Stack` everywhere?

Reply:
@muvaf that is fine on my end. Note that we followed this naming strategy in the other repos as well https://github.com/crossplane/stack-aws-sample/blob/c1e7a30b31d6a1760410800a62f557e7c60c396a/.registry/app.yaml#L1

Reply:
@muvaf just confirmed with @rathpc that your proposed update here looks good :+1: AWS will need to be updated also but GCP is good to go I believe. Thanks!

Reply:
Thanks for bringing this up @muvaf, good eye! Please do make this update :)

Reply:
Done:
https://github.com/crossplane/stack-aws-sample/pull/16
https://github.com/crossplane/stack-azure-sample/pull/14